src/app/public/styles/themes/modern/_box.scss

```diff
@@ -0,0 +1,7 @@
+@import '../../variables-public';
+
+.sky-theme-modern {
+  .sky-box {
```

Comment:
A question for maybe for my own education ... does the "box" convention have inherit meaning? All its doing here is adding a rounded corner. Just curious why its called "box" ?

Reply:
It's intended to be a generic container that can be adorned with various elevations; currently it only has rounded corners, but we may add to it down the road. Also, "box" is what design has been calling it and has in the specs for the new theme.